assets/src/scss/blocks/PostsListBlock.scss

```diff
@@ -0,0 +1,143 @@
+.posts-list-query {
+  .wp-block-post-template.is-layout-flow li {
+    width: 100% !important;
+  }
+
+  .wp-block-columns.is-layout-flex {
+    gap: $sp-2x;
+  }
+
+  .wp-block-post-featured-image {
+    flex-basis: 30%;
+
+    img {
+      @include medium-and-up {
+        height: 180px;
+      }
+
+      @include large-and-up {
+        height: 210px;
+      }
+      width: 100% !important;
+      object-fit: cover;
+    }
+  }
+
+  .wp-block-group.is-layout-flow {
+    flex-basis: 70%;
+  }
+
+  .wp-block-post-title {
+    font-weight: $font-weight-bold;
+    color: var(--grey-900);
+    font-family: var(--font-family-tertiary);
+    font-size: $font-size-l--font-family-primary;
+    margin-top: calc($sp-x + 1px);
+  }
+
+  .wp-block-post-excerpt {
+    margin-top: calc(5px - $sp-2x);
+
+    &__excerpt {
+      width: 85%;
+      -webkit-line-clamp: 3;
+
+      @include medium-and-up {
+        -webkit-line-clamp: 2;
+        width: 100%;
+      }
+
+      @include large-and-up {
+        -webkit-line-clamp: 3;
+      }
+
+      @include x-large-and-up {
+        -webkit-line-clamp: 3;
+      }
+
+      margin-bottom: $sp-1;
+      display: -webkit-box;
+      -webkit-box-orient: vertical;
+      overflow: hidden;
+      font-size: var(--font-size-m--font-family-secondary);
+    }
+  }
+
+  .is-layout-flow > .is-layout-flow {
+    display: flex;
+    gap: $sp-x;
+    font-size: calc($font-size-xxs--font-family-primary - 2px);
+    color: var(--grey-600);
+    font-family: var(--font-family-paragraph-secondary);
+  }
+
+  .wp-block-post-author-name {
+    font-size: calc($font-size-xxs--font-family-primary - 2px);
```

Comment:
This looks hard to follow up!

Reply:
Based on the tokens we have, we don't have a token for `14px`, so I used the `calc()` to subtract `2px` from the token we have for `16px` (based of your recommendation to use tokens). I could add the values in comments to make it clearer.